src/mavlink-router/endpoint.h

```diff
@@ -185,6 +185,14 @@ class UdpEndpoint : public Endpoint {
     struct sockaddr_in sockaddr;
 
 protected:
+
+    void _reschedule_write();
+    int _write_internal_buffer();
+    bool _write_scheduled;
+
+    Timeout* _write_schedule_timer = nullptr;
+    const unsigned int _nagle_buf_max_packet_size, _nagle_max_timeout_ms;
```

Comment:
nagle is a misnomer, that algorithm has no timers but relies on ACKS which is not applicable -- suggest just dropping the term it is slightly misleading

Reply:
Will do